index.js

```diff
@@ -1,40 +1,100 @@
 const fs = require('fs-extra');
 
-
 const getAllFilteredItems = (path, isAllDirectory, isAllFiles) => {
 
-    return (fs.readdirSync(path) || []).filter(item => {
-        const isDirectory = fs.statSync(`${path}/${item}`).isDirectory();
```

Comment:
can you please change your indent to 4 spaces?

Reply:
You mean tab instead of line space?

Reply:
@yogevmalcaym no there are indent options use 4 spaces